osquery/logger/plugins/tests/buffered_tests.cpp

```diff
@@ -215,19 +215,19 @@ TEST_F(BufferedLogForwarderTests, test_async) {
   EXPECT_CALL(*runner, send(ElementsAre("foo"), "result"))
       .WillOnce(Return(Status(0)));
   runner->logString("foo");
-  std::this_thread::sleep_for(1.5 * kLogPeriod);
+  std::this_thread::sleep_for(5 * kLogPeriod);
```

Comment:
We can't eliminate the `sleep_for` on this line without changing the semantics of the test. Part of what we are checking here is that the successfully "sent" log is purged by the run that occurs during this `sleep_for`, and the log added on line 225 is processed in the next runs during the `sleep_for` on 226.

We can remove line 230 though, as it was only relevant when the sleeps were timed just long enough to allow the desired number of periods to pass.